docs/topics/home.xml

```diff
@@ -50,7 +50,6 @@
                     <a href="koans.md" description="Programming exercises to get you familiar with Kotlin">Koans</a>
                     <a href="https://hyperskill.org/join/fromdocstoJetSalesStat?redirect=true&amp;next=/tracks/18" description="Kotlin Basics track on JetBrains Academy">JetBrains Academy</a>
                     <a href="advent-of-code.md" description="Code puzzles in idiomatic Kotlin">Advent of Code</a>
-                    <a href="https://play.kotlinlang.org/hands-on/overview" description="Complete long-form tutorials to fully grasp a technology">Hands-on tutorials</a>
```

Comment:
Let's keep these links (to the Hands-ons overview) and revert changes here and below.
We will investigate some cases during the onboarding research